examples/column_transformer.py

```diff
@@ -104,21 +65,22 @@ def fit(self, x, y=None):
         return self
 
     def transform(self, posts):
-        features = np.recarray(shape=(len(posts),),
-                               dtype=[('subject', object), ('body', object)])
+        # construct object dtype array with two columns
+        # first column = 'subject' and second column = 'body'
+        features = np.empty(shape=(len(posts), 2), dtype=object)
```

Comment:
As we are going to be in the "column" namespace, where we support pandas dataframes, should we use a pandas dataframe in this example, rather than a object array?